scripts/build_benchmark_configs.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+
+# Helper script to build multiple configurations of a benchmark.  Patches the
+# board description files to enable and disable different combinations of
+# hardware features.  Also patches include paths to keep track of the original
+# locations.
+
+set -eo pipefail
+
+SCRIPTPATH=$(dirname $(readlink -f $0))
+SDKPATH=$(realpath ${SCRIPTPATH}/../sdk)
```

Comment:
Stale?

Reply:
Yes, that’s left over from when I was doing ${sdk} substitution, but I don’t think we need it so I removed the complicated jq expression.